### DISCUSSION TITLE
License not compatible

### DISCUSSION THREAD

**Discussion by ajm8580** [aclap-dev/video-downloadhelper#953] [Help and Q&A]
### Your question

Hello,
I just bought a license using Brave to download audios of videos and then I found out 5min later that I cannot use this app on youtube because it is blocked. It suggested me to use firefox, but now it says that I own a chrome license and not a firefox one and that I cannot use it on firefox....
Is there a way to use the same license on both browser? I find it amazing that it is not working on both browser.
If not, can I swap it? (give up the chrome license and get a firefox one)
Many thanks

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: fr
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"5c0ef8174e55464e"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 9,
    "successfull_dl": 2
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You need to click the Wiki link that appears at the top of every page of this forum.  After you read the main page, look down the table of contents on the right side of the page.  Click the chapter for `I purchased the wrong premium` & follow the instructions there.

**Reply by ajm8580** [reply]
Hi GrampaWildWilly. Many thanks for your answer, it solved it!
